src/esp/bindings_js/index.js

```diff
@@ -5,17 +5,25 @@ import VRDemo from "./modules/vr_demo";
 import "./bindings.css";
 
 Module.preRun.push(() => {
```

Comment:
At this point, it will be better to move this to its own class which has functions like preloadMp3d etc.